package.json

```diff
@@ -27,6 +27,7 @@
     "@nestjs/graphql": "7.0.1",
     "@nestjs/platform-express": "7.0.1",
     "apollo-server-express": "2.11.0",
+    "dotenv": "^8.2.0",
```

Comment:
Pin :)

Reply:
Oh, I forgot that :) Thanks for the reminder (Y)